### DISCUSSION TITLE
Is there a way to detect loitering?

### DISCUSSION THREAD

**Discussion by Cold-Lemonade** [blakeblackshear/frigate#6868] [Ask A Question]
Is there a way for Frigate to detect a person loitering in a specific zone? For example, one of my cameras looks out at the stairs to my front door. Is there a way for Frigate to determine that someone has been on the stairs for, say, 30 seconds?

**Comment by NickM-27** [comment]
Yes, this is possible via the mqtt events topic

**Reply by Cold-Lemonade** [reply]
What's the general idea to get this working? I mean how do I set up Frigate to distinguish loitering from someone just passing through the zone? I use mqtt to notify Home Assistant that someone has entered a zone, but how do I get Frigate to tell Home Assistant that someone has been hanging out in the zone for a certain amount of time?

**Reply by NickM-27** [reply]
You could use motionless count, stationary, etc. There is a lot of information available https://docs.frigate.video/integrations/mqtt#frigateevents
